astropy/io/fits/file.py

```diff
@@ -83,6 +83,7 @@ class _File(object):
     @deprecated_renamed_argument('clobber', 'overwrite', '2.0')
     def __init__(self, fileobj=None, mode=None, memmap=None, overwrite=False,
                  cache=True):
+        from http.client import HTTPResponse
```

Comment:
It's not a big deal at all for me to do the portable thing, so I can just update it to use `six.moves` for now.